foundationsync/indico_foundationsync.py

```diff
@@ -231,8 +239,19 @@ def fetch_rooms(self, connection, room_name=None):
 
             # Update room data
             room = self._update_room(room, room_data, room_attrs)
+            # Remove all existing full_access acl entries before adding the new ones
+            # otherwise changing the room owner/manager-group wont remove the old one
+            for manager in room.get_manager_list():
+                room.update_principal(manager, full_access=False)
```

Comment:
do not remove the current room owner here if there is no new owner, otherwise the current owner loses access even though he's still listed as the owner